specs/DesignDiscussions/InputLabelPosition.md

```diff
@@ -0,0 +1,105 @@
+# Input Label Position
+
+## Overview
+Right now position of the label is always above to the input field. We want to provide more options to card authors to position the label.
+Now, labels can also be place inline with the input. It can provide better experience and more flexibity to card authors in different scenarios.
+
+## New Schema Changes in Adaptive Card
+
+Introducing an optional new styling property in AC input fields schema which will allow developers to define `position` of the label with respect to input field. It can be both `inline` and `above`. Default would be existing `above`.
+	*Note*: When card width is small due to browser resize etc, horizontal view can shift to vertical view of input fields.
```

Comment:
can you elaborate on horizontal and vertical view? How do we determine what's too small? we can wrap and still maintain label to input field host config ratio, for example, in this case, we don't necessarily need to switch to different view.

Reply:
yeah makes sense. We can still maintain inline view. I will remove this line.